notebooks/258-decidiffusion-image-generation/258-decidiffusion-image-generation.ipynb

```diff
@@ -0,0 +1,1321 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #44.        gc.collect()
</pre><p>Please align garbage collection across different modules. It should be called inside the function with object removal.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1379/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
aligned